rules/rules-reviewed/eap8/eap7/hibernate-search-6.1.windup.xml

```diff
@@ -12,6 +12,7 @@
         </dependencies>
         <targetTechnology id="eap" versionRange="[8,)" />
         <targetTechnology id="hibernate" versionRange="[6.1,)" />
+        <targetTechnology id="quarkus" versionRange="[3,)" />
```

Comment:
Good catch. I was wondering about the `javax` to `jakarta` changes... Should we capture those rules somehow for quarkus migration?